tests/sentry/search/events/test_builder.py

```diff
@@ -1203,6 +1208,9 @@ def test_run_query_with_events_per_aggregates(self):
         assert data["tpm"] / 60 == data["tps"]
 
     def test_failure_rate(self):
+        for string in ["transaction.status", "internal_error", "ok"]:
```

Comment:
should `transaction` ,`transaction.status`, `internal_error`, `ok` all be in `SHARED_TAG_STRINGS`? cc @jan-auer

Reply:
I spoke with @jan-auer about this and he thinks we should go ahead with this.. I posted all the tag names/values for transactions in an earlier comment